src/com/goide/intentions/GoMoveToStructInitializationIntention.java

```diff
@@ -60,12 +60,34 @@ public boolean isAvailable(@NotNull Project project, Editor editor, @NotNull Psi
   private static Data getData(@NotNull PsiElement element) {
```

Comment:
getData method became overcomplicated. In my opinion, such methods should look like this:

```
val data
data.somePieceOfData1 = gatherSomePieceOfData_1()
data.somePieceOfData2 = gatherSomePieceOfData_2()
data.somePieceOfData3 = gatherSomePieceOfData_3()
return data
```

For now it looks like endless `if, if, != null, return null`. It's barely readable and hard to edit.